ivy_tests/test_ivy/test_frontends/test_tensorflow/test_random.py

```diff
@@ -118,3 +119,70 @@ def test_tensorflow_shuffle(
         value=values[0],
         seed=seed,
     )
+
+
+# random_stateless_uniform
+@handle_frontend_test(
+    fn_tree="tensorflow.random.stateless_uniform",
+    shape=helpers.dtype_and_values(
+        available_dtypes=tuple([ivy.int64, ivy.int32]),
+        min_value=1,
+        max_value=5,
+        min_num_dims=1,
+        max_num_dims=1,
+    ),
+    seed=helpers.dtype_and_values(
+        available_dtypes=tuple([ivy.int64, ivy.int32]),
+        min_value=0,
+        max_value=10,
+        min_num_dims=1,
+        max_num_dims=1,
+        min_dim_size=2,
+        max_dim_size=2,
+    ),
+    minval=helpers.ints(min_value=0, max_value=3),
+    maxval=helpers.ints(min_value=4, max_value=10),
+    dtype=helpers.get_dtypes("numeric", full=False),
+    test_with_out=st.just(False),
+)
+def test_tensorflow_stateless_uniform(
+    shape,
+    seed,
+    minval,
+    maxval,
+    dtype,
+    frontend,
+    test_flags,
+    fn_tree,
+    on_device,
+):
+    if (
+        "complex" in dtype[0]
+        or "int8" in dtype[0]
+        or "int16" in dtype[0]
+        or "uint" in dtype[0]
+    ):
+        return
+
+    shape_input_dtypes, shape = shape
+    seed_input_dtypes, seed = seed
+
+    if len(seed[0]) < 2:
+        return
```

Comment:
thanks, i added that because of the `helpers.dtype_and_values` problem as stated in my reply.

But after changing the parameter and using `shape` , I can omit the array length check now.